test/e2e/framework.go

```diff
@@ -368,13 +374,17 @@ func collectClusterInfo() error {
 			podV6NetworkCIDR: podV6NetworkCIDR,
 			gwV4Addr:         gwV4Addr,
 			gwV6Addr:         gwV6Addr,
+			os:               node.Status.NodeInfo.OperatingSystem,
+		}
+		if node.Status.NodeInfo.OperatingSystem == "windows" {
+			clusterInfo.windowsNodes[node.Name] = true
+			clusterInfo.windowsNodeIdxes = append(clusterInfo.windowsNodeIdxes, nodeIdx)
```

Comment:
let's use "Indexes", I think "Idxes" is ugly :)

Reply:
Updated. :)